bugbug/bug_features.py

```diff
@@ -132,6 +135,18 @@ def __call__(self, bug):
         return any(bug['creator_detail']['email'].endswith(domain) for domain in ['@mozilla.com', '@mozilla.org'])
 
 
+class delta_request_merge(object):
+    def __call__(self, bug):
+        for change in bug['history']:
```

Comment:
Nit: Call this `history` and `ind_change` `change`, to keep the same variable names as other parts of the project.